src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java

```diff
@@ -121,8 +123,8 @@ public static boolean RequestHTTPS(
         // json has odd characters at the front, trim them to make valid json
         int jsonStartIndex = fileStr.indexOf("{");
 
-        int notFound  = -1;
-        if(jsonStartIndex == notFound){
+        int notFound = -1;

```

Comment:
My previous comment on formatting seems to be resolved in this commit -- please move the formatting change for the `int notFound  = -1;` line to the original commit where it was added